ge/views.py

```diff
@@ -100,6 +101,21 @@ def edit_librarydata(request: HttpRequest, item_id: int) -> HttpResponse:
     return render(request, "ge/edit_item.html", {"form": form})
 
 
+@login_required(login_url="/login/")
+def add_fund(request: HttpRequest) -> HttpResponse:
+    if request.method == "POST":
+        form = LibraryDataEditForm(request.POST)
+        if form.is_valid():
+            new_fund = form.save()
+            messages.success(request, "Fund added successfully.")
+            return redirect(
```

Comment:
This is good, putting the user on the new record for immediate editing as needed.